client/blocks/upe/upe-deferred-intent-creation/payment-processor.js

```diff
@@ -52,6 +52,7 @@ const getStripeElementOptions = () => {
 			applePay: 'never',
 			googlePay: 'never',
 		},
+		layout: 'accordion',
```

Comment:
This can be the default setting for all implementations, since we just display more than one method when SPE is enabled anyway.